artifacthub-repo.yml

```diff
@@ -0,0 +1,17 @@
+# Artifact Hub repository metadata file
+#
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: 825ae023-f622-4de9-a6be-a4a041cf8fdf
+owners: # (optional, used to claim repository ownership)
+  - name: jaroslav.fedor
+    email: jaroslav.fedor@solarwinds.com
+ignore: # (optional, packages that should not be indexed by Artifact Hub)
+  - name: swi-k8s-opentelemetry-collector
```

Comment:
Why should this be ignored?

Reply:
Showing old repository might be confusing, and we definitely dont want customer to install that.